packages/core/src/StateMachine.ts

```diff
@@ -94,10 +94,7 @@ export class StateMachine<
 
   public implementations: MachineImplementationsSimplified<TContext, TEvent>;
 
-  /** @internal */
-  public __xstatenode: true = true;
```

Comment:
Technically it is a breaking change - one that we don't have to make. We shouldn't freely remove things unless there is a very strong reason to do that.